core/drivers/fabric-driver/server/fabric-code.ts

```diff
@@ -77,6 +77,8 @@ const getNetworkGateway = async (networkName: string): Promise<Gateway> => {
 async function invoke(
     query: query_pb.Query,
     networkName: string,
+    funName: string,
```

Comment:
I'd prefer we call this `funcName` to match our naming conventions elsewhere in this repository.

Reply:
Done